src/containers/Chat/Chat.tsx

```diff
@@ -115,11 +119,32 @@ const Chat: React.SFC<ChatProps> = ({ contactId }) => {
   if (loading) return <Loading />;
   if (error) return <p>Error :(</p>;
 
+  const getConversations = () => {
+    const allConversations: any = client.readQuery({
```

Comment:
Why are we doing this? The query is fired in this component itself to get the conversations. We don't need to read it from the cache.